src/Controls/src/Core/HandlerImpl/VisualElement/VisualElement.cs

```diff
@@ -24,15 +24,11 @@ public partial class VisualElement
 				[nameof(IViewHandler.ContainerView)] = MapContainerView,
 			};
 
-		static CommandMapper<IView, IViewHandler> ControlsViewCommandMapper = new(ViewHandler.ViewCommandMapper)
-		{
-			[nameof(IView.Focus)] = MapFocus,
-		};
-
 		internal static void RemapForControls()
 		{
 			ViewHandler.ViewMapper = ControlsVisualElementMapper;
-			ViewHandler.ViewCommandMapper = ControlsViewCommandMapper;
+
+			ViewHandler.ViewCommandMapper.ReplaceMapping<VisualElement, IViewHandler>(nameof(IView.Focus), MapFocus);
```

Comment:
I think this needs to be `ModifyMapping` and then you call the `action` part of `ModifyMapping` here https://github.com/dotnet/maui/pull/15040/files#diff-0b23daf01dc61f5761c5fbf2bf67bed116f20788abc248c741a4984ef24ffbedL483

Reply:
This ReplaceMapping is a new method in net8 so we can rename. This is why I was adding the When suffix to the methods when I added them. ReplaceMappingWhen is a tiny bit more descriptive.

Should we rename? In a new PR?

Reply:
Aw ok I'm following now!

yea I think this is all fine.